$OpenBSD: patch-zathura_shortcuts_c,v 1.1 2014/11/30 23:14:48 edd Exp $

Fix incorrect use of booleans and glib that trash the stack.
https://git.pwmt.org/?p=zathura.git;a=commit;h=8d71a755d648c5856ef91d214bfdb46925f6c2f0

--- zathura/shortcuts.c.orig	Sun Nov  9 22:50:01 2014
+++ zathura/shortcuts.c	Sat Nov 29 18:40:34 2014
@@ -859,9 +859,11 @@ sc_search(girara_session_t* session, girara_argument_t
   const unsigned int num_pages = zathura_document_get_number_of_pages(zathura->document);
   const unsigned int cur_page  = zathura_document_get_current_page_number(zathura->document);
   GtkWidget *cur_page_widget = zathura_page_get_widget(zathura, zathura_document_get_page(zathura->document, cur_page));
-  bool nohlsearch, first_time_after_abort, draw;
+  bool nohlsearch, first_time_after_abort;
+  gboolean draw;
 
-  nohlsearch = first_time_after_abort = draw = false;
+  nohlsearch = first_time_after_abort = false;
+  draw = FALSE;
   girara_setting_get(session, "nohlsearch", &nohlsearch);
 
   if (nohlsearch == false) {
